testsuite/integration/clustering/pom.xml

```diff
@@ -604,9 +605,9 @@
             </build>
         </profile>
 
-        <!-- Run cluster tests with standalone-ha.xml profile with Infinispan Server provisioned -->
+        <!-- Run cluster tests with standalone-ha.xml profile with reuseForks == false -->
         <profile>
-            <id>ts.clustering.cluster.ha-infinispan-server.profile</id>
+            <id>ts.clustering.cluster.ha-noReuseForks.profile</id>
```

Comment:
The rest of the id-s are standardised using dash-separated-names instead of CamelCase. Do you want to rename these to minimise confusion?

Reply:
Done.